cmake/pcl_targets.cmake

```diff
@@ -662,6 +662,8 @@ endmacro()
 ###############################################################################
 # Write a report on the build/not-build status of the subsystems
 macro(PCL_WRITE_STATUS_REPORT)
+  message(STATUS "PCL build with following flags:")
```

Comment:
Were these intended to be included?

Reply:
Yeah. I added them to check the flags (for the -Werror issue), but then thought: this would be nice to have anyways in the status report